scripts/src/server.sh

```diff
@@ -130,6 +132,14 @@ if [ -z "$CONTAINER_NAME" ]; then
   CONTAINER_NAME="${CONTAINER_NAME/:/-}-ssh"
 fi
 
+if [[ ! -f "${SSH_KEY_FILE}" ]]; then
+  if [[ ! -f "${DEFAULT_SSH_RSA}" ]]; then
+    echo "SSH not set up! Configure SSH on your system."
+    exit 1
+  fi
+  echo "Provided SSH key file ${SSH_KEY_FILE} does not exist. Using default ${DEFAULT_SSH_RSA}"
```

Comment:
Because you set the EDRSA file as the default `SSH_KEY_FILE` above, in most cases we will see this printed line, which is kind of annoying. In particular, if I don't supply a `-k` argument, I would be confused by this message saying something about `id_ed25519.pub`. There's no perfect solution here because someone might have multiple key pairs for different reasons. Preferably you just print out one line at the end of this whole if block that says "using SSH key file ${}". 

I would say the RSA should be default, and if it doesn't exist then choose EDRSA if it exists. If both exist, it will be RSA by default.